hawkular-wildfly-monitor/src/main/java/org/hawkular/agent/monitor/inventory/Operation.java

```diff
@@ -16,18 +16,12 @@
  */
 package org.hawkular.agent.monitor.inventory;
 
-public abstract class Operation<RT extends ResourceType<?, ?, ?, ?>> extends NamedObject {
+public abstract class Operation extends NamedObject {
 
-    private final RT resourceType;
     private String operationName;
 
-    public Operation(ID id, Name name, RT resourceType) {
```

Comment:
why remove this? Its possible that given an operation, we'll want to know what resource type it was associated with. Without this link we lose that context.

same with ResourceConfigurationPropertyType